src/main/java/com/aws/greengrass/deployment/DeploymentService.java

```diff
@@ -203,7 +201,9 @@ protected void startup() throws InterruptedException {
         // Reset shutdown signal since we're trying to startup here
         this.receivedShutdown.set(false);
         reportState(State.RUNNING);
-        loadDeploymentQueueFromConfig(); // Load any deployments from queue during previous shutdown
+
+        // Clear any queue persistence data (removed feature).
+        this.config.lookup(DEPLOYMENT_QUEUE_TOPIC).remove();
```

Comment:
let's use find so we don't create the topic

Reply:
fixed